apps/tup-cms/src/taccsite_cms/templates/snippets/css-news-hacks.html

```diff
@@ -1,15 +0,0 @@
-<style class="css-news-hacks">
-    /* To hide byline of an article by another outlet */
-    /* NOTE: Requires author add "external" tag to the article */
-    /* TODO: Figure out how to show external author that user enters */
-    .has-blog-tag-external .byline {
-        display: none;
-    }
```

Comment:
Yesterday, CMD Lead Writer and CMD Website Admin agreed to not hide this anymore.

I deleted actual snippet and updated CMS data to ensure all now visible "bylines" look good.

Reply:
|actual poster | unknown poster<br /><sup>(previously, this was hidden)</sup> |
| - | - |
| ![Actual Poster](https://github.com/TACC/tup-ui/assets/62723358/e5dd8ee1-91b6-4be7-9916-fea66cb096fd) | ![Unknown Poster](https://github.com/TACC/tup-ui/assets/62723358/600ce498-ac16-4c92-bced-b18b079b164f) |